inscriptions/ISic0249.xml

```diff
@@ -174,70 +179,20 @@
                 <ab>
-                    <lb n="1"/>
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <supplied reason="lost">hic requi</supplied>esc
-                    <supplied reason="lost">it in pace</supplied>
-                    <lb n="2"/>
-                    <supplied reason="lost">bone memo</supplied>riae A
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <supplied reason="lost">anno</supplied>
-                    <lb break="no" n="3"/>
-                    <supplied reason="lost">rum plus</supplied>
-                    minus XLV
-                    <supplied reason="lost">die</supplied>
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <lb n="4"/>
-                    <supplied reason="lost">
-                        <expan>
-                            <abbr>s</abbr>
-                            <ex>is</ex>
-                        </expan>
-                    </supplied>
-                    A
-                    <expan>
-                        <ex>u</ex>
-                        <abbr>g</abbr>
-                        <ex>usti</ex>
-                    </expan>
-                    <expan>
-                        <abbr>ind</abbr>
-                        <ex>ictione</ex>
-                    </expan>
-                    XV im
-                    <supplied reason="lost">
-                        <expan>
-                            <abbr>p</abbr>
-                            <ex>eratore</ex>
-                        </expan>
-                        <expan>
-                            <abbr>d</abbr>
-                            <ex>omino</ex>
-                        </expan>
-                        <expan>
-                            <abbr>n</abbr>
-                            <ex>ostro</ex>
-                        </expan>
-                    </supplied>
-                    <lb n="5"/>
-                    <supplied reason="lost">H</supplied>eraclio piissimo
-                    <lb n="6"/>
-                    <expan>
-                        <abbr>s</abbr>
-                        <ex>emper</ex>
-                    </expan>
-                    <expan>
-                        <abbr>A</abbr>
-                        <ex>ugusto</ex>
-                    </expan>
-                    .
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/><supplied reason="lost">hic requi</supplied>esc<supplied reason="lost">it in pace</supplied>
+                    <lb n="2"/><supplied reason="lost">bone memo</supplied>riae A<gap reason="lost" extent="unknown" unit="character"/><supplied reason="lost">anno</supplied>
+                    <lb break="no" n="3"/><supplied reason="lost">rum plus</supplied> minus <num value="45">XLV</num> <supplied reason="lost">die</supplied> <gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
We're going for `<lb n="3" break="no"/>` from now on I think

Reply:
I can change those everywhere but agreed about future edits

Reply:
Yes, that's ok? I remember you prefer this way...

Reply:
ah, sorry, yes, that's fine!